README.md

```diff
@@ -71,8 +71,6 @@ include the Java runtime and do not require it to be installed separately:
 The Java JAR [`logisim-evolution-<version>-all.jar`](https://github.com/logisim-evolution/logisim-evolution/releases)
 is also available and can be run on any system with a supported Java runtime installed.
 
-[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/logisim-evolution)
-
```

Comment:
Why you want to delete this line? The site with the Snap package is still online.